calcom/cal.com
# fix: Issue with Email Notifications - Time Format

## What does this PR do?

This PR fixes a part of issue i.e., the time format showing 24h with am/pm as reported by user. 

Fixes #12407

## Requirement/Documentation

NA

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## How should this be tested?

Try creating a meeting and check the email. It should display am/pm only in case of 12h format.
